ttnn/cpp/ttnn/operations/ccl.hpp

```diff
@@ -38,9 +38,38 @@ struct ExecuteAllGather {
     }
 };
 
+struct ExecuteLineAllGather {
```

Comment:
this should be moved to a separate file

The expected structure is 

operations/ccl/line_all_gather <-- files related to this op go here
operations/ccl/all_gather
operations/ccl/reduce_scatter